Drivers/Inc/Minions.h

```diff
@@ -27,7 +27,8 @@ typedef enum {
      REGEN_SW,
      CRUZ_ST,
      IGN_1,
-     IGN_2
+     IGN_2,
+     NUM_SWITCHES,
```

Comment:
I'm not a huge fan of this, I would rather use a macro for NUM_SWITCHES, but it's fine if nobody else has a problem with it.